.github/workflows/reusable-e2e-tests-run.yml

```diff
@@ -64,7 +64,7 @@ jobs:
       - run: docker compose -f docker-compose.yml run --rm api migrate-database
 
       # start necessary containers
-      - run: docker compose -f docker-compose.yml up -d api frontend pdf print browserless database docker-host http-cache mail reverse-proxy
+      - run: docker compose -f docker-compose.yml up -d api frontend pdf print browserless database docker-host http-cache mail pg-admin reverse-proxy
```

Comment:
Why do we need pgadmin in CI?

Reply:
The reverse-proxy container checks whether the configured endpoints (servers) are accessible. If not, this leads to an error.